### PR TITLE
Document how to use legacy and new Operator together

### LINKED ISSUE #10: Document how to use legacy and new Operator together
### Description

This is a workaround for the missing CRs.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by vmuzikar**
Closes #10

### CODE REVIEW

**Review by ahus1 — COMMENTED** [review]
@vmuzikar - I've read the text, and I suppose I understand what it is trying to achieve. I'm not familiar with the old operator, and I haven't tested this setup, so I only "comment" here. 

If you think that I should be involved more in testing this / fully reviewing this, please let me know.

**Comment by vmuzikar** [comment]
@ahus1 Thanks for the review! It'd be great if someone could try the steps in case I overlooked something. Doesn't mean it needs to be necessarily you but I'd really  appreciate it. :)

In any case, I'd like to have this merged rather sooner than later to give people time to migrate and use the new Operator in case the missing CRs are blocking them.

**Comment by hmlnarik** [comment]
I am unsubscribing myself from the review as this is not my area of expertise and I rely on @ahus1 doing better job here than I would.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Client CRs with Quarkus distribution
+
+This repository contains examples with a temporary workaround for the missing Client CR in the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator).
+
+## Motivation
+
+To provide the best experience, the new Operator will use a new approach to manage Keycloak resources, such as Realms, Clients and Users. This approach will leverage the [new storage architecture](https://www.keycloak.org/2022/07/storage-map.html) and future immutability options, making the CRs the declarative single source of truth. In comparison to the [legacy Operator](https://github.com/keycloak/keycloak-operator), this will bring high robustness, reliability and predictability to the whole solution.
+
+However, since the new storage is still WIP and currently is lacking some features like the options for immutability that are crucial for the CRs, the new Operator is currently missing those CRs for managing Keycloak resources. The missing CRs will be added once the new store has support for it.
```
Comment (hmlnarik) [inline README.md]: Same comment as in https://github.com/keycloak/keycloak-web/pull/329#pullrequestreview-1088584195 applies here

Comment (vmuzikar) [reply README.md]: Good catch, thanks! Updated the PR.

**Review by pedroigor — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
I reviewed the text and it reveals the necessary ideas and concepts to a user to achieve this. I also see that the functionality is in line what we planned in terms of functionality for in the storage team. I didn't test the approach though, and will leave that to those who are familiar with it in the CN team.

**Pull request merged**